app/Services/AWS.php

```diff
@@ -119,6 +119,7 @@ public function deleteImage($path)
         // any exception that is thrown while trying to delete and just returns true.
         // see: \Illuminate\Filesystem\FilesystemAdapter::delete().
         // So we check if the file exists first and then try to delete it.
+        dd('sending ' . $path . ' to storage');
         if (Storage::exists($path)) {
```

Comment:
I'm seeing something strange here - I did this `dd` along with a `dd` in `namespace Illuminate\Filesystem\Filesystem.php` which `Storage::exists()` sends to. 

I expect to get the `dd('sending ' . $path . ' to storage');` message first when I hit the `DELETE` endpoint but instead, I am getting the `dd` message I placed in `Filesystem.php` with this `$path`: `"/home/vagrant/DoSomething.org/rogue/bootstrap/cache/services.php"`

I don't know why it is hitting that before the `dd('sending ' . $path . ' to storage');` - @sbsmith86 is this the caching you were mentioning perhaps?

Reply:
It seems like this is getting called before anything in the function `deleteImage` but I'm not sure where this is getting called? cc @DoSomething/team-bleed

Reply:
@chloealee maybe try changing that dd in `Filesystem.php` to a log message in case that is getting hit on something you're not expecting, then you can still see the info on the hit you were expecting

Reply:
oh that is a great idea, @katiecrane ! So I did `dump` instead and got this:
```
"exists /home/vagrant/DoSomething.org/rogue/bootstrap/cache/services.php"
"exists /home/vagrant/DoSomething.org/rogue/bootstrap/cache/routes.php"
"exists /home/vagrant/DoSomething.org/rogue/bootstrap/cache/routes.php"
"sending to exists 198-923df5957838355206574f72d5520f0f-1527198297.jpeg"
{"success":{"code":200,"message":"Post deleted."}}
```

Seems like it doesn't actually go to `exists` function from `deleteImage()`?